nnstreamer/tensor_trainer/tensor_trainer_nntrainer.cc

```diff
@@ -339,31 +354,31 @@ void NNTrainer::NNTrainerTrain::getRunStats() {
   ml_logd("<leave>");
 }
 
-NNTrainer::InputTensorsInfo::InputTensorsInfo(unsigned int _total_num_samples,
-                                              unsigned int _num_inputs,
-                                              unsigned int _num_labels,
-                                              unsigned int _tensors_size[]) :
+SubPlugin::TensorsQueue::TensorsQueue(unsigned int _num_of_samples,
+                                      unsigned int _num_of_inputs,
+                                      unsigned int _num_of_labels,
+                                      unsigned int _tensors_size[]) :
   queue_front(0),
   queue_rear(0),
   push_count(0),
   pop_count(0),
-  total_num_samples(_total_num_samples),
-  num_inputs(_num_inputs),
-  num_labels(_num_labels) {
+  num_of_samples(_num_of_samples),
+  num_of_inputs(_num_of_inputs),
+  num_of_labels(_num_of_labels) {
 
   ml_logd("<called>");
   const unsigned int min_queue_size = 30;
   queue_size =
-    (_total_num_samples > min_queue_size) ? min_queue_size : _total_num_samples;
+    (_num_of_samples > min_queue_size) ? min_queue_size : _num_of_samples;
   ml_logd("queue_size:%d", queue_size);
-  tensor_data.reserve(queue_size);
+  queue.reserve(queue_size);
 
   unsigned int idx = 0, i = 0;
-  for (i = 0; i < num_inputs; i++) {
+  for (i = 0; i < num_of_inputs; i++) {
```

Comment:
If ```idx == i```, you don't need to iterate each to copy.

Reply:
Thank you.

Reply:
When I determine that the debugging log is not needed, I will replace it with ```memcpy()```. Thank you.